b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/manageUserSection/otherComponents/editUserComponent.tsx

```diff
@@ -130,7 +130,8 @@ export default function EditUserComponent(prop: EditUserComponentProps) {
             successTypeDialog(toaster, "Changes Saved Successfully", "User details edited successfully.");
             onClose();
         } else {
-            warningTypeDialog(toaster, "Roles not Updated", "Error occured while updating the roles. Try again.");
+            warningTypeDialog(toaster, "Roles not Properly Updated",
```

Comment:
`Roles not Properly Updated` -> `Roles are not properly updated`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c4229e64c82f57a16a579a24259eafab0e6cc56f